src/main/java/io/confluent/idesidecar/restapi/models/ConnectionSpec.java

```diff
@@ -53,6 +54,15 @@ public record CCloudConfig(
 
   }
 
+  @Schema(description = "Configuration for local cluster")
+  public record LocalConfig(
+      @JsonProperty(value = "schema-registry-uri") String schemaRegistryUri
+  ) {
+    public String schemaRegistryUri() {
+      return this.schemaRegistryUri;
+    }
+  }
```

Comment:
Why do we want to use a URL here, rather than a port or multiple ports? If it's a _local connection_, then should we assume the schema registry is also running locally?

Reply:
We are using the URI for the kafka-rest.  See application.yml  `kafkarest-uri`.  This is consistent with a similar option in the section of applciation.yml file.

Should we allow the user to point to an SR that is not running locally but on a different host? Why not? I don't see any strong benefits or significant downsides to disallowing it. @rhauch

Reply:
Sounds good -- I just wanted to be intentional here.